lv_objx/lv_list.h

```diff
@@ -119,6 +119,13 @@ bool lv_list_remove(const lv_obj_t * list, uint32_t index);
 /*=====================
  * Setter functions
  *====================*/
+ 
+/**
+ * Make a single button selected in the list, deselect others, should be called in list btns call back.
+ * @param btn pointer to the currently pressed list btn object
+ */
+void lv_list_set_btn_single_selected(lv_obj_t *btn);
```

Comment:
Maybe the public API should be an option that would cause `lv_list_set_btn_single_selected` to be called right before the list button callback.

Reply:
Well, if optional, I think it will need an ext->single_selected_mode.
What do you think?

Reply:
For me `lv_list_set_btn_single_selected` also sounds like an API function which makes the list to enable only one selection automatically.  So I also vote for `ext->single_selected_mode`. In my opinion if `ext->single_selected_mode` is set then `lv_list_add` can automatically call `lv_btn_set_toggle` to make the button selectable.

Reply:
I agree, update in latter time.
> In my opinion if ext->single_selected_mode is set then lv_list_add can automatically call lv_btn_set_toggle to make the button selectable.
```lv_btn_set_toggle```
It looks expatiatory sometimes，but most flexibility.

Reply:
I will review the code again later.